<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectValueDialog</class>
 <widget class="QDialog" name="ObjectValueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>267</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="7" column="0" colspan="2">
    <widget class="KDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="KComboBox" name="m_comboBox"/>
   </item>
   <item row="1" column="1">
    <widget class="KIntSpinBox" name="m_intSpinBox">
     <property name="prefix">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="m_doubleSpinBox"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="m_lineEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="KTextEdit" name="m_textEdit"/>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="m_checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="6">
    <widget class="QLabel" name="m_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>kdialogbuttonbox.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_intSpinBox</tabstop>
  <tabstop>m_buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ObjectValueDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ObjectValueDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
